addons/web/static/src/views/utils.js

```diff
@@ -239,6 +239,20 @@ export function processMeasure(measure) {
     return measure === "__count__" ? "__count" : measure;
 }
 
+/**
+ * Remove user specific info from the context
+ * @param {Object} context
+ * @returns {Object}
+ */
+export function removeContextUserInfo(context) {
```

Comment:
Not sure, when is this necessary?

Reply:
@aab-odoo Hello!
In our case (Knowledge) it is to store the context (without the author user info) of an `embedded view` in a JSON object in the `body` `html_field`, so that it can be rendered again by anybody reading an article. And as that function was already written twice in spreadsheet, I'm proposing that we put it somewhere in `web`

https://github.com/odoo/enterprise/pull/32783/files